src/NuGet.Core/NuGet.Commands/NuGet.Commands.csproj

```diff
@@ -3,10 +3,12 @@
 
   <PropertyGroup>
     <Description>Complete commands common to command-line and GUI NuGet clients</Description>
-    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+    <TargetFrameworks Condition="'$(IsVsixBuild)' != 'true'">netstandard1.3;net45</TargetFrameworks>
```

Comment:
i talked to @emgarten abut this. it turns out that the netcore project system starts treating everything relative to the file which has the $(TargetFramework) property, and that's why we can't move it to common.project.props. Justin, please correct me if i am wrong.